server/game/cards/02_SHD/upgrades/VambraceFlamethrower.ts

```diff
@@ -0,0 +1,27 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { UpgradeCard } from '../../../core/card/UpgradeCard';
+import { Location, RelativePlayer, WildcardCardType } from '../../../core/Constants';
+
+export default class VambraceFlamethrower extends UpgradeCard {
+    protected override getImplementationId() {
+        return {
+            id: '6471336466',
+            internalName: 'vambrace-flamethrower',
+        };
+    }
+
+    public override setupCardAbilities() {
```

Comment:
Need to add the vehicle attach restriction and add a test for it. See any of the lightsabers for an example.